latest/docs-ref-conceptual/release-notes-azure-cli.md

```diff
@@ -16,6 +16,63 @@ ms.assetid: ce0428f7-0a59-4e72-9237-d907b171af51
 
 # Azure CLI 2.0 release notes
 
+## October 9, 2017
+
+Version 2.0.19
+
+### Core
+
+* Azure Stack: handle adfs authority url with a trailing slash
+
+### Appservice
+
+* webapp: added generic update with new command: 'az webapp update'
+
+### Batch
+
+* Updated to Batch SDK 4.0.0.
+* Updated --image option of VirtualMachineConfiguration to support ARM image references in addition to publish:offer:sku:version.
+* Now supports the new CLI extension model for Batch Extensions commands - support for old component model has been removed.
+
+### Batchai
+
+* Initial release of Batch AI module.
+
+### Keyvault
+
+* Fixed Key Vault authentication issue when using ADFS on Azure Stack. https://github.com/Azure/azure-cli/issues/4448
+
+### Network
+
+* `application-gateway address-pool create`: `--server` argument is not optional to allow creation of empty address pools.
```

Comment:
I can't tell what this actually is. Did it fix a bug where creating empty pools was possible, or did it make the `--server` argument optional, or did it change behavior so an empty `--server` value creates an empty pool?

Reply:
It makes the param `--server` optional.
It closes https://github.com/Azure/azure-cli/issues/4533